express/blocks/blog-posts/blog-posts.css

```diff
@@ -0,0 +1,278 @@
+.blog main div.section:has(.blog-posts) {
+    padding: 0;
+    margin-top: 64px;
```

Comment:
please don't add margin or anything to the sections.

Reply:
I couldn't find the alternative to replace the padding of the section right now.